doc/encodings.rdoc

```diff
@@ -137,15 +137,16 @@ returns the \Encoding of the concatenated string, or +nil+ if incompatible:
 A Ruby String object has an encoding that is an instance of class \Encoding.
 The encoding may be retrieved by method String#encoding.
 
-The default encoding for a string literal is the script encoding
-(see Encoding@Script+encoding):
+The default encoding for a string literal is the script encoding;
+see {Script Encoding}[https://docs.ruby-lang.org/en/master/encodings_rdoc.html#label-Script+Encoding].
```

Comment:
Why can't we use a relative `rdoc-ref` here?

Reply:
I could not make it work.

Reply:
Using `encodings.rdoc@Script+Encoding` works for me, which generates the text "see Script Encoding at encodings.rdoc". Alternatively, we can use `(Script Encoding)[rdoc-ref:encodings.rdoc@Script+Encoding]`, which might be better because it omits the filename and generates the text "see Script Encoding".

Reply:
Fixed.

Reply:
I don't see this as fixed.